xlators/features/quiesce/src/quiesce.c

```diff
@@ -2541,6 +2541,9 @@ init(xlator_t *this)
     this->private = priv;
     ret = 0;
 out:
+    if (priv) {
```

Comment:
```suggestion
    if (ret) {
```

in success case, we shouldn't bother freeing `priv` and if ret is =-1, then GF_FREE has a null check for priv, hence no need to check for priv